### PR TITLE
Update mini cart test to include increased max quantity scenario

### PR DESCRIPTION
**Pull request by veljkho**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Updated existing mini cart test to include check for max quantity.
Updated existing cart block test to improve logging and naming.

Closes # [773](https://github.com/woocommerce/woocommerce-quality/issues/773) and [774 ](https://github.com/woocommerce/woocommerce-quality/issues/774) in Woo Quality

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch
2. Execute two tests locally:
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/mini-cart.spec.js`
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/cart-block.spec.js`
3. Make sure they pass

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

**Review by adimoldovan — CHANGES_REQUESTED** [review]
Thanks @veljkho for these improvements. 

Can you please also address the following: 
- the tests are dependent on each other. The second test is failing if it's running standalone, because the first one is basically its setup.
- the pages are not cleaned-up and not unique per run, making a second run use the page of the first. You can easily test this one by running both tests and then only running the second one. It should have failed, because the first one didn't run to create its page, but instead it passes, because it uses a page left from a previous run.

**Comment by veljkho** [comment]
Hey @adimoldovan 
Thanks for reviewing all my PRs today, appreciate it.

> the tests are dependent on each other. The second test is failing if it's running standalone, because the first one is basically its setup.

Sure, they are now single tests as per the new commit.

> the pages are not cleaned-up and not unique per run, making a second run use the page of the first. You can easily test this one by running both tests and then only running the second one. It should have failed, because the first one didn't run to create its page, but instead it passes, because it uses a page left from a previous run.

Yes, added commit to make it unique.

File: plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block.spec.js [hunk 1]
```diff
@@ -81,18 +82,9 @@ test.describe( 'Cart Block page', () => {
 		} );
 	} );
 
-	test.beforeEach( async ( { context } ) => {
-		// Shopping cart is very sensitive to cookies, so be explicit
-		await context.clearCookies();
-	} );
-
-	test( 'can see empty cart block', async ( { page } ) => {
+	test( 'can create and see empty cart block', async ( { page } ) => {
```
Comment (adimoldovan) [inline plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block.spec.js]: I'm not sure if the test name is relevant to what's actually checked. I think you wanted to keep the other test name?

File: plugins/woocommerce/tests/e2e-pw/tests/shopper/mini-cart.spec.js [hunk 2]
```diff
@@ -110,46 +111,21 @@ test.describe( 'Mini Cart block page', () => {
...
-		} );
-	} );
-
-	test.beforeEach( async ( { context } ) => {
-		// shopping cart is very sensitive to cookies, so be explicit
-		await context.clearCookies();
 	} );
 
-	test( 'can see empty mini cart', async ( { page } ) => {
+	test( 'can create and see empty mini cart', async ( {
```
Comment (adimoldovan) [inline plugins/woocommerce/tests/e2e-pw/tests/shopper/mini-cart.spec.js]: Same here, I believe you meant to keep the other test name.

**Review by adimoldovan — COMMENTED** [review]
Thanks @veljkho for the updates. It tested well for me.

Can you please re-check the test names as per my comments?

File: plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block.spec.js (hunk 1 above)
Comment (veljkho) [reply plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block.spec.js]: That's a miss, I was too fast. Good catch!

File: plugins/woocommerce/tests/e2e-pw/tests/shopper/mini-cart.spec.js (hunk 2 above)
Comment (veljkho) [reply plugins/woocommerce/tests/e2e-pw/tests/shopper/mini-cart.spec.js]: I updated both test names to reflect better what's testing there. Thanks again!

**Review by adimoldovan — APPROVED** [review]

**Pull request merged**